tests/components/zha/test_channels.py

```diff
@@ -159,4 +171,203 @@ def test_channel_registry():
     for (cluster_id, channel) in registries.ZIGBEE_CHANNEL_REGISTRY.items():
         assert isinstance(cluster_id, int)
         assert 0 <= cluster_id <= 0xFFFF
-        assert issubclass(channel, channels.ZigbeeChannel)
+        assert issubclass(channel, base_channels.ZigbeeChannel)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5051600)